test/Sentry.Tests/SentryHttpFailedRequestHandlerTests.cs

```diff
@@ -265,4 +347,62 @@ public void HandleResponse_Hint_Response()
             hint.Items[HintTypes.HttpResponseMessage].Should().Be(response);
         }
     }
+
+    [Fact]
+    public void HandleResponse_ExceptionHasStackTrace()
```

Comment:
suggestion: Add a "scenario-part" in the middle of the test name, to be compliant with out `CONTRIBUTING.md`.

Maybe something like

```suggestion
    public void HandleResponse_Capture_ExceptionHasStackTrace()
```